Makefile

```
@@ -8,11 +8,7 @@ BUILD_META=-build$(shell date +%Y%m%d)
 ORG ?= rancher
 TAG ?= v3.20.2$(BUILD_META)
 
-ifeq ($(ARCH),s390x)
-K3S_ROOT_VERSION ?= v0.10.0-rc.0
-else
-K3S_ROOT_VERSION ?= v0.9.1
-endif
+K3S_ROOT_VERSION ?= v.0.10.1
```

```suggestion
K3S_ROOT_VERSION ?= v0.10.1
```

Typos happen :smile: (and this one was my doing)